discussion.md

```diff
@@ -0,0 +1,54 @@
+#Discussion 
+
+##1. Summary of PCA paper
+The question that PCA asks is "Is there another basis which is a linear combination of the original basis that best re-expresses our dataset?" (It is quoted from the paper but I think it best expresses what the analysis does) — @Sarah Majors 
```

Comment:
Whenever expressing quoting, you can use **> Text to be quoted.**

> This is a quoting example.

Reply:
Quoted text comes only in new line.